src/main/ConfigDialog.scala

```diff
@@ -94,4 +94,7 @@ class PythonMenu extends JMenu("Python") {
   add("Configure").addActionListener{ _ =>
     new ConfigEditor(App.app.frame, PythonExtension.config).setVisible(true)
   }
+  add("Pop-out Interpreter").addActionListener{ _ =>
```

Comment:
I don't love the name `Pop-out Interpreter`.  Most NetLogo-ish would be `Open Python Command Center`, but you could talk me into something like `Interactive Python Console`.  What do you think?

Reply:
My gut is that the Python extension is for people who want to make netlogo models but get frustrated enough with doing things in vanilla netlogo that they turn to a more familiar language like Python to do some heavy lifting. To me these people would be more familiar with the non-netlogo terminology than they would be the command center. I forgot what the command center was for a minute when I first read this honestly.